ntbs-service/Pages/Notifications/Edit/Episode.cshtml

```diff
@@ -44,7 +57,8 @@ ViewData["Title"] = "Notification - Hospital Details";
         <nhs-form-group nhs-form-group-type=@ConsultantGroupState v-bind:class="{ 'nhsuk-form-group--error': hasError}">

```

Comment:
Does this `hasError` want to go as well?
There's an error in the console on this page ATM, so it might be relevant:
```
Property or method "hasError" is not defined on the instance but referenced during render. Make sure that this property is reactive, either in the data option, or for class-based components, by initializing the property. See: https://vuejs.org/v2/guide/reactivity.html#Declaring-Reactive-Properties.

found in

---> <ValidateInput>
       <Root>
```

Reply:
Not all Views are fixed to work without hasError binding, I will create a ticket and change them all today (1-2 hours work)